server/server.js

```diff
@@ -19,94 +19,66 @@ app.use('/rooms/:id', express.static(path.join(__dirname, '../client/dist')));
 // Decided not to use a router, considering there are only two routes.
```

Comment:
I really love these notes that you leave for yourself and the reviewer (the one re: cors, the one about the router, etc.). this is a great habit. i'm going to steal it.

Reply:
haha...you need to thank Alex. Those are his original notes. But I agree they are super helpful.